tests/unit/amo/reducers/test_addons.js

```diff
@@ -153,32 +153,13 @@ describe(__filename, () => {
 
   it('sets `isMozillaSignedExtension` to `false` when not declared', () => {
     const addon = createFakeAddon({
-      files: [{ is_mozilla_signed_extension: false }],
+      file: { is_mozilla_signed_extension: false },
     });
 
     const state = addons(stateWithLang, loadAddon({ addon, slug: addon.slug }));
     expect(getAddonByID(state, addon.id).isMozillaSignedExtension).toBe(false);
   });
 
-  it('sets `isMozillaSignedExtension` to `false` without files', () => {
-    const addon = createFakeAddon({ files: [] });
-
-    const state = addons(stateWithLang, loadAddon({ addon, slug: addon.slug }));
-    expect(getAddonByID(state, addon.id).isMozillaSignedExtension).toBe(false);
-  });
-
-  it('sets `isMozillaSignedExtension` to `true` when any file declares it', () => {
-    const addon = createFakeAddon({
-      files: [
-        { is_mozilla_signed_extension: false },
-        { is_mozilla_signed_extension: true },
-      ],
-    });
-
-    const state = addons(stateWithLang, loadAddon({ addon, slug: addon.slug }));
-    expect(getAddonByID(state, addon.id).isMozillaSignedExtension).toBe(true);
-  });
```

Comment:
I think `file` is nullable (or should be) so the three test cases where I left a comment (including this one) should stay with `file: null` or something. Unless, there is something I am missing?

Reply:
The type information above only allows `file: AddonFileType` (not `file: AddonFileType|null`) so that can't happen, or that's what I thought?